src/ui.button/index.stories.js

```diff
@@ -21,29 +21,15 @@ export default {
 };
 
 const DefaultTemplate = (args) => ({
-  components: { UButton },
+  components: { UButton, UIcon },
   setup() {
     const slots = getSlotNames(UButton.name);
 
     return { args, slots };
   },
   template: `
     <UButton v-bind="args">
-      <template v-for="(slot, index) of slots" :key="index" v-slot:[slot]>
```

Comment:
Why this one was removed? This code is needed to real time test slots in storybook. Please return back.